MS5611.cpp

```diff
@@ -7,7 +7,7 @@
 //     URL: https://github.com/RobTillaart/MS5611
 //
 //  HISTORY:
-//
+//  0.3.2.1 2021-12-25  Update oversampling timings to reduce time spent waiting
```

Comment:
this is an illegal version number (but I will fix that when release is comming)

Reply:
Fixed!